extmod/modbluetooth_nimble.c

```diff
@@ -25,13 +25,13 @@
  * THE SOFTWARE.
  */
 
+#if MICROPY_PY_BLUETOOTH && MICROPY_BLUETOOTH_NIMBLE
```

Comment:
These guards usually go after the standard py/ includes, so that they can (in principle) be configured by a header file like mpconfigport.h.  Is there a reason to move it up here?